src/Symfony/Component/OptionsResolver/Exception/ExceptionInterface.php

```diff
@@ -16,6 +16,6 @@
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
  */
-interface ExceptionInterface
+interface ExceptionInterface extends \Throwable
```

Comment:
This interface, like the equivalent ones throughout the other components (about 20 of them), is meant to be a marker, nothing more.

So, I would not extend `\Throwable` here. But if we decide to do so, would it make sense to do it on all our markers?

Reply:
This marker is for exceptions though, shouldn't it guarantee it's used on exception classes only?

Reply:
Deprecating implies to find a name for the replacement in addition to bother our users, I don't think it's worth the hassle.

Reply:
Regarding name, it could be {Component}ExceptionInterface. E.g. ConsoleExceptionInterface.

Reply:
I think it makes sense to extend Throwable everywhere to semantically indicate that it is for exceptions. No need for deprecations as this should not break anything.

Reply:
Ok at least somebody is with me, so I've applied this change in rest of the components as well.